### PR TITLE
Added remove accessors stage

### PR DESCRIPTION
**Pull request by leerichard42**

### CODE REVIEW

File: lib/removeUnusedAccessors.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    var meshes = gltf.meshes;
+    var skins = gltf.skins;
+    var animations = gltf.animations;
+
+    // Build hash of used accessors by iterating through meshes, skins, and animations
+    if (defined(meshes)) {
+        for (var meshId in meshes) {
+            if (meshes.hasOwnProperty(meshId)) {
+                if (defined(meshes[meshId].primitives)) {
+                    var primitives = meshes[meshId].primitives;
```
Comment (pjcozzi) [inline lib/removeUnusedAccessors.js]: This line can go above the `if` statement, so the `if` statement would be:
```javascript
if (defined(primitives)) {
```

File: lib/removeUnusedAccessors.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                    var length = primitives.length;
+                    for (var i = 0; i < length; i++) {
+                        if (defined(primitives[i].attributes)) {
+                            var attributes = primitives[i].attributes;
```
Comment (pjcozzi) [inline lib/removeUnusedAccessors.js]: Similar comment here.

File: lib/removeUnusedAccessors.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                            for (var attributeId in attributes) {
+                                if (attributes.hasOwnProperty(attributeId)) {
+                                    var primitiveAccessorId = attributes[attributeId];
+                                    usedAccessorIds[primitiveAccessorId] = true;
+                                }
+                            }
+                        }
+                        if (defined(primitives[i].indices)) {
```
Comment (pjcozzi) [inline lib/removeUnusedAccessors.js]: Same comment.

File: lib/removeUnusedAccessors.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+                var skinAccessorId = skins[skinId].inverseBindMatrices;
+                usedAccessorIds[skinAccessorId] = true;
+            }
+        }
+    }
+    if (defined(animations)) {
+        for (var animationId in animations) {
+            if (animations.hasOwnProperty(animationId)) {
+                if (defined(animations[animationId].parameters)) {
+                    var parameters = animations[animationId].parameters;
```
Comment (pjcozzi) [inline lib/removeUnusedAccessors.js]: Same comment.

**Comment by pjcozzi** [comment]
Looks great!  Just those minor style comments.

**Comment by leerichard42** [comment]
Updated!

**Comment by pjcozzi** [comment]
Nice work!

**Pull request merged**